docs/userguide/configuration.rst

```diff
@@ -2969,6 +2970,48 @@ For more on prefetching, read :ref:`optimizing-prefetch-limit`
 
     Tasks with ETA/countdown aren't affected by prefetch limits.
 
+.. setting:: worker_enable_prefetch_count
+
+``worker_enable_prefetch_count``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.3
```

Comment:
This should be in 5.4 as this is a new feature.

Reply:
Why?
This flag does not break compatibility, and defacto leaves the same behavior due to defaults.

Reply:
Patch releases are reserved for bug fixes. There is no bug to fix here.
Minor releases are for new features that don't break backward compatibility.

Reply:
Fix pushed